WordPress/src/main/java/org/wordpress/android/support/main/ui/SupportActivity.kt

```diff
@@ -55,20 +63,45 @@ class SupportActivity : AppCompatActivity() {
                             isLoggedIn = isLoggedIn,
                             showAskTheBots = optionsVisibility.showAskTheBots,
                             showAskHappinessEngineers = optionsVisibility.showAskHappinessEngineers,
+                            showNetworkDebugging = networkTrackingState.showNetworkDebugging,
+                            isNetworkTrackingEnabled = networkTrackingState.isTrackingEnabled,
+                            networkTrackingRetentionInfo = getRetentionInfoText(
+                                networkTrackingState.retentionPeriod
+                            ),
                             versionName = WordPress.versionName,
                             onBackClick = { finish() },
                             onLoginClick = { viewModel.onLoginClick() },
                             onHelpCenterClick = { viewModel.onHelpCenterClick() },
                             onAskTheBotsClick = { viewModel.onAskTheBotsClick() },
                             onAskHappinessEngineersClick = { viewModel.onAskHappinessEngineersClick() },
-                            onApplicationLogsClick = { viewModel.onApplicationLogsClick() }
+                            onApplicationLogsClick = { viewModel.onApplicationLogsClick() },
+                            onNetworkTrackingToggle = { viewModel.onNetworkTrackingToggle(it) },
+                            onViewNetworkRequestsClick = { viewModel.onViewNetworkRequestsClick() },
                         )
                     }
                 }
             }
         )
     }
 
+    private fun getRetentionInfoText(period: NetworkRequestsRetentionPeriod): String {
+        val periodString = getRetentionPeriodDisplayString(period)
+        return getString(R.string.network_requests_retention_info, periodString)
+    }
+
+    private fun getRetentionPeriodDisplayString(period: NetworkRequestsRetentionPeriod): String {
+        return when (period) {
+            NetworkRequestsRetentionPeriod.ONE_HOUR ->
+                getString(R.string.network_requests_retention_one_hour)
+            NetworkRequestsRetentionPeriod.ONE_DAY ->
+                getString(R.string.network_requests_retention_one_day)
+            NetworkRequestsRetentionPeriod.ONE_WEEK ->
+                getString(R.string.network_requests_retention_one_week)
+            NetworkRequestsRetentionPeriod.FOREVER ->
+                getString(R.string.network_requests_retention_until_cleared)
+        }
+    }
```

Comment:
⛏️ Very minor readability nitpick

```suggestion
private fun getRetentionPeriodDisplayString(period: NetworkRequestsRetentionPeriod): String {
    return getString(getRetentionPeriodStringRes(period))
}

private fun getRetentionPeriodStringRes(period: NetworkRequestsRetentionPeriod): Int {
    return when (period) {
        NetworkRequestsRetentionPeriod.ONE_HOUR -> R.string.network_requests_retention_one_hour
        NetworkRequestsRetentionPeriod.ONE_DAY -> R.string.network_requests_retention_one_day
        NetworkRequestsRetentionPeriod.ONE_WEEK -> R.string.network_requests_retention_one_week
        NetworkRequestsRetentionPeriod.FOREVER -> R.string.network_requests_retention_until_cleared
    }
}
```